std/experimental/allocator/package.d

```diff
@@ -605,6 +605,8 @@ unittest
     assert(a == [0, 0, 0]);
 }
 
+import std.experimental.allocator.gc_allocator: isGCSafeAllocator;
```

Comment:
I just read the entire history of 314 - pretty scary :fearful: 
so I basically should do `import std.experimental.allocator.gc_allocator`, because that doesn't leak?